x-pack/plugins/apm/public/components/app/service_overview/index.tsx

```diff
@@ -38,36 +23,26 @@ import { useTimeRange } from '../../../hooks/use_time_range';
 export const chartHeight = 288;
 
 export function ServiceOverview() {
```

Comment:
Maybe we could call this function `ServiceOverviewRouter` since it's going to render multiple versions of the service overview? WDYT

Reply:
Since it's not really a router, what if we call it `ServiceOverviewWrapper` or something similar?

Reply:
But this is not a `Wrapper` either right?! Maybe `ServiceOverviewPage` but we don't use `page` at all. I don't know, if you find a better name go for it otherwise leave it as it is.

Reply:
That's true. I can merge this and see if we come up with a better name and change it in the other mobile APM issues.